test/import/FSHImporter.Profile.test.ts

```diff
@@ -451,7 +459,14 @@ describe('FSHImporter', () => {
         const result = importText(input);
         const profile = result.profiles.get('ObservationProfile');
         expect(profile.rules).toHaveLength(1);
-        assertFixedValueRule(profile.rules[0], 'status', new FshCode('final'));
+        const expectedCode = new FshCode('final');
+        expectedCode.location = {
+          startLine: 6,
+          startColumn: 20,
+          endLine: 6,
+          endColumn: 25
+        };
```

Comment:
Oh.  Another idea to consider... if you actually made that chained method in `FshEntity` like this instead:
```ts
withLocation(location: TextLocation | [number, number, number, number]): this {
  if (Array.isArray(location)) {
    this.location = {
      startLine: location[0],
      startColumn: location[1],
      endLine: location[2],
      endColumn: location[3]
    };
  } else {
    this.location = location;
  }
  return this;
}
```

Then these unit tests would be less annoying since the above seven lines could be shortened to:
```ts
const expectedCode = new FshCode('final').withLocation(6, 20, 6, 25);
```

I don't know about you, but I think I really like that.

Reply:
Of course, assuming we also need to have a space for file, I guess that array might contain a string as well.  So maybe:
```ts
withLocation(location: TextLocation | [string, number, number, number, number]): this {
  if (Array.isArray(location)) {
    this.location = {
      file: location[0],
      startLine: location[1],
      startColumn: location[2],
      endLine: location[3],
      endColumn: location[4]
    };
  } else {
    this.location = location;
  }
  return this;
}
```
?